config/software/omnibus-toolchain.rb

```diff
@@ -40,9 +40,13 @@
 # ruby core tools
 dependency "ruby"
 
-dependency "nokogiri"
+# Projects such as Chef Server, Supermarket, Chef Manage and Chef Backend use berkshelf for the build time installation of gems.
+# Moving it to toolchain avoids installing a ton of deps into our packages.
+if linux? || mac_os_x?
```

Comment:
```suggestion
if linux? || macos?
```

that's the real name of the helper now

Reply:
thanks. i'll update the branch when my adhoc build finishes